src/utils/arrayUtils.ts

```diff
@@ -0,0 +1,33 @@
+function slice<T>(array: T[], start: number, end: number): T[] {
```

Comment:
What does this do opposed to `Array.prototype.slice`?

Reply:
great point, replaced with the native one